### DISCUSSION TITLE
Change App Favicon

### DISCUSSION THREAD

**Discussion by hci-ikahera** [Budibase/budibase#11444] [Help] [answered]
Hello everyone,
Anyone here know how to change favicon for Budibase App? Is it possible?
Thank you 🙏🏼

Regards,
Ika

**Comment by ConorWebb96** [comment, marked as answer]
Hey @hci-ikahera,

You should be able to do this in the branding section `builder/portal/settings/branding`. This is a premium feature. This can be found and done in the highlighted field shown in the screenshot below;

<img width="1277" alt="Screenshot 2023-08-07 at 08 18 42" src="https://github.com/Budibase/budibase/assets/126772285/5a3c497e-6f9a-4b94-a14d-37356bf448ca">

I hope this helps.

**Reply by hci-ikahera** [reply]
Oh okay. Thank you @ConorWebb96 🙏🏼
